src/app/devices/sol/poweralert/poweralert.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'app-poweralert',
+  templateUrl: './poweralert.component.html',
+  styleUrls: ['./poweralert.component.scss']
+})
+export class PoweralertComponent implements OnInit {
```

Comment:
this should be renamed to  `PowerAlertComponent`

Reply:
done